From 521f66239cbacea63483f25e2dcb8cea3c5b4a06 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 12 Jan 2021 21:24:37 +0200
Subject: [PATCH] [SW-0] preboot: raise comment precision

Improve comments precision regarding the externs of _start and
__stack_top.
Those are depending the execution environment, so specify it in comment.
No functional changes.

Change-Id: I9aa244eeaace79ebaccccb14134f5aabf7da62c0
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/include/preboot.h b/include/preboot.h
index 102eb59..da01ae2 100644
--- a/include/preboot.h
+++ b/include/preboot.h
@@ -60,8 +60,8 @@
 	HL_DEBUG_HW_DBG_CARD,
 };
 
-extern void *_start;		/* Start of preboot region */
-extern void *__stack_top;	/* End of preboot region */
+extern void *_start;		/* Start of ppboot/preboot region */
+extern void *__stack_top;	/* End of ppboot/preboot region */
 
 static inline u8 regmap_readb(u32 offset)
 {
